rasa_nlu/registry.py

```diff
@@ -67,6 +67,15 @@
 # necessary components in the right order. They also implement
 # the preexisting `backends`.
 registered_pipeline_templates = {
+    "pretrained_embeddings": [
```

Comment:
I think we should  remove the old templates and define a mapping from the old template names to the new ones in `pipeline_template` or in `config.py`, but that's open for discussion

Reply:
I agree that would be cleaner, but since we will remove these anyway in next major release I think it's not worth making the code more elegant